HISTORY.rst

```diff
@@ -2,7 +2,26 @@
 History
 =======
 
+3.52.0 (2022-02-03)
+-------------------
+
+Critical Changes
+
+* We fixed a bug where the ``Run Task`` and ``Run Task Class`` robot keywords could throw an error like ``AttributeError: module 'robot.api.logger' has no attribute 'log'`` in rare cases (#3053)
```

Comment:
This belongs under Issued Closed, not Critical Changes (critical changes is for when we changed something in a backwards incompatible way)